src/utils/item.js

```diff
@@ -46,3 +46,23 @@ export const comparePurchaseUrgency = (data) => {
 	const dataWithDays = appendDaysToData(data);
 	return sortedMultiArr(dataWithDays).map(sortByNextDateAlphabetical).flat();
 };
+
+export const getUrgency = (urgencyGroup) => {
```

Comment:
Wow!! At first I was like what is this object with anonymous functions in them, and then I realized these are callbacks to be passed in the filter function! Where did you learn it or did you come up with it! Very cool!!

Reply:
I definitely saw it somewhere before! Just wanted to try it out here.